app/src/main/res/layout/activity_main.xml

```diff
@@ -206,6 +206,58 @@
         </android.support.v7.widget.CardView>
     </LinearLayout>
 
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:clipToPadding="false"
+        android:gravity="center"
+        android:orientation="horizontal">
+
+        <android.support.v7.widget.CardView
```

Comment:
When you'll rebase, just copy paste your CardView with the Information CardView. The `LinearLayout` is already there.